src/components/media-link/media-link.stories.mdx

```diff
@@ -0,0 +1,82 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { mediaArgTypes } from '../../objects/media/demo/arg-types.js';
+import { avatarArgTypes } from '../avatar/demo/arg-types.js';
+import template from './media-link.twig';
+const mediaLinkArgTypes = {
+  ...mediaArgTypes,
```

Comment:
Because the template extends the Media Object directly, we can pretty much just import its `argTypes` as is. This isn't true for Avatar (see other comment).